src/executables/util_cs.c

```diff
@@ -3296,17 +3328,41 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
 				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
 				     &applied_final_lsa);
+	  if (error != NO_ERROR)
+	    {
+	      goto print_applyinfo_usage;
+	    }
```

Comment:
```suggestion
	  if (error != NO_ERROR)
	    {
	      goto print_applyinfo_usage;
	    }
```
I think this code is also not necessary.

Reply:
I think same problem above comment what your comment is written.

[above comment](https://github.com/CUBRID/cubrid/pull/3374#discussion_r813744394)